Dockerfile

```diff
@@ -1,16 +1,19 @@
 FROM ubuntu:14.04
 
-MAINTAINER Christoph Wiechert <wio@psitrax.de>
-MAINTAINER https://github.com/henszey
-
 RUN apt-get update
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get install -y nodejs
 
 RUN mkdir /app
 ADD . /app/
 
+RUN groupadd -g 1024 ebrowser && \
+  useradd -u 1024 -g ebrowser ebrowser && \
+  chown -R ebrowser:ebrowser /app
+
 WORKDIR /app
-EXPOSE 8000
+EXPOSE 8080
```

Comment:
what was your motivation to change the port ?

Reply:
I don't remember (this is an old diff). The port change was surely not necessary.